scalding-core/src/main/scala/com/twitter/scalding/reducer_estimation/InputSizeReducerEstimator.scala

```diff
@@ -32,8 +32,8 @@ object InputSizeReducerEstimator {
     Common.inputSizes(info.step) match {
       case Nil =>
         LOG.warn("InputSizeReducerEstimator unable to estimate reducers; " +
-          "cannot compute size of:\n - " +
-          Common.unrollTaps(info.step).filterNot(_.isInstanceOf[Hfs]).mkString("\n - "))
+          "cannot compute size of one of (is it a non hfs tap?):\n - " +
```

Comment:
this is back to `cannot compute the size of(is it a non Hfs tap): Tap1, Tap2` right? cause we can have a JDBC tap in there with a HDFS based tap and we'd just skip the jdbc one.